src/test/ui/rfc-2086-irrefutable_let_patterns/allow_irrefutable_let_patterns.stderr

```diff
@@ -0,0 +1,8 @@
+warning: the feature `irrefutable_let_patterns` has been stable since 1.32.0 and no longer requires an attribute to enable
+  --> $DIR/allow_irrefutable_let_patterns.rs:12:12
+   |
+LL | #![feature(irrefutable_let_patterns)]
```

Comment:
Remove this line from `allow_irrefutable_let_patterns.rs` so you can remove this file.